index.html

```diff
@@ -5,7 +5,8 @@
   <meta charset="UTF-8" />
   <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
   <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-  <title>Vite App</title>
+  <link href="data:image/x-icon;base64,AAABAAEAEBAQAAAAAAAoAQAAFgAAACgAAAAQAAAAIAAAAAEABAAAAAAAgAAAAAAAAAAAAAAAEAAAAAAAAAAAAAAARoJAACdKIwAxXC0A////AE+RSQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAREREREQAAABDMyIiJAAAAEFEMkQkAAAAQUNDRCQAAABBREM0JAAAAEVBQUQ0RERERUQREzRDMyJFVVERNEFEMkVVVREUQUNDRERERERBRENDJAAAAEVBQUQ0AAAARUQREzQAAABFVVERNAAAAEVVVREUAAAAREREREQAAAD8AAAA/AAAAPwAAAD8AAAA/AAAAPwAAAAAAAAAAAAAAAAAAAAAAAAAAD8AAAA/AAAAPwAAAD8AAAA/AAAAPwAA" rel="icon" type="image/x-icon" />
```

Comment:
To powinno być podobnie do tego jak jest teraz. Czyli ikonka pobrana i wrzucona do plików, a tutaj link zamiast base64

Reply:
```suggestion
  <link rel="icon" type="image/svg+xml" href="/favicon.svg" />
```

i ta ikona favicon.svg powinna zostać podmieniona z tej, którą mamy aktualnie.

Reply:
```suggestion
  <link href="/images/logo.jpg" rel="icon" type="image/x-icon" />
```

Reply:
Albo coś podobnego

Reply:
ten base64 to jest coś z binarką zwiazanego, raczej nieistotne :D